plyer/platforms/linux/camera.py

```diff
@@ -0,0 +1,49 @@
+import subprocess
+from plyer.facades import Camera
+
+
+class LinuxCamera(Camera):
```

Comment:
Again, name it appropriately. It's GStreamer's functionality, which might be available even on Windows (not sure) :)